common/test/acceptance/pages/studio/overview.py

```diff
@@ -600,6 +602,14 @@ def make_exam_timed(self, hide_after_due=False):
         self.q(css=".action-save").first.click()
         self.wait_for_ajax()
 
+    def make_subsection_hidden_after_due_date(self):
+        """
+        Sets a subsection to be hidden after due date.
+        """
+        self.q(css='input[value=hide_after_due]').first.click()
+        self.q(css=".action-save").first.click()
+        self.wait_for_ajax()
```

Comment:
Yeah, just a potential speed gain. I don't think it matters a huge amount here, it seems to be an established pattern.